graylog2-server/src/main/java/org/graylog2/indexer/datanode/ProxyRequestAdapter.java

```diff
@@ -20,7 +20,8 @@
 import java.io.InputStream;
 
 public interface ProxyRequestAdapter {
-    record ProxyRequest(String method, String path, InputStream body, String hostname) {}
+    record ProxyRequest(String method, String path, InputStream body, String hostname,
+                        jakarta.ws.rs.core.MultivaluedMap<String, String> queryParameters) {}
```

Comment:
This could be added to import, no conflicting MultivalueMap in this class.

Reply:
fixed, thanks!